README.md

```diff
@@ -3,9 +3,25 @@
 
 This repository houses all documentation for Jellyfin available at [jellyfin.org](https://docs.jellyfin.org/) and written in markdown.
 
+# Contributing
+
+The site is built with [DocFX](https://dotnet.github.io/docfx/) using [DocFX Flavored Markdown](https://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html). See the content section [here](https://dotnet.github.io/docfx/tutorial/docfx_getting_started.html) for a quick tutorial on DocFX.
+
+Since the site is mostly written with simple Markdown files, the easiest and fastest way to contribute is to just edit the source files directory on GitHub. For example, you could edit this README page by going to its [edit page](https://github.com/jellyfin/jellyfin-docs/edit/master/README.md) on GitHub.
+
+Editing directly on GitHub provides a feature to preview your changes for the current document, but if you want to see your changes within the context of the actual website or make more advanced changes to the site, you will need to run a copy of the site locally.
+
+To run the site locally, you can start by following the instructions to install DocFx as a [command line tool](https://dotnet.github.io/docfx/tutorial/docfx_getting_started.html#2-use-docfx-as-a-command-line-tool). Once installed, you can run the following command from the repository root directory.
+
+```bash
+docfx --serve
```

Comment:
should we include a git clone https://github.com/jellyfin/jellyfin-docs command?